src/js/video.js

```diff
@@ -333,6 +333,31 @@ videojs.registerTech = Tech.registerTech;
  **/
 videojs.use = middlewareUse;
 
+/**
+ * An object that can be returned by a middleware to signify
+ * that the middleware is being terminated.
+ *
+ * @type {object}
+ * @memberOf {videojs}
+ * @property {object} middleware.TERMINATOR
+ **/
+// Object.defineProperty is not available in IE8
+if (!browser.IS_IE8 && Object.defineProperty) {
+  Object.defineProperty(videojs, 'middleware', {
+    value: {},
+    writeable: false,
+    enumerable: true
+  });
+
+  Object.defineProperty(videojs.middleware, 'TERMINATOR', {
+    value: TERMINATOR,
+    writeable: false,
+    enumerable: true
+  });
+} else {
+  videojs.middleware.TERMINATOR = TERMINATOR;
```

Comment:
this will fail because `videojs.middleware` isn't an object.
Should be `videojs.middleware = {TERMINATOR}`